the-name-card/src/main.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.scss';
+import { Landing, Margin } from './components';
+
+ReactDOM.createRoot(document.getElementById('root')!).render(
+	<React.StrictMode>
+		<Margin>
+			<Landing />
```

Comment:
you should be rendering `App.tsx` here, not `Landing`, your `Margin` and `Landing` components should be in `App` instead

Reply:
yeap mb, should be putting the navigation in for this. Will add that in before resolving this